tsl/src/compression/compression.c

```diff
@@ -430,13 +430,22 @@ compress_chunk(Oid in_table, Oid out_table, int insert_options)
 	}
 
 	RowCompressor row_compressor;
+	/* Reset sequence is used for resetting the sequence without
+	 * having to fetch the existing sequence for an individual
+	 * sequence group. This is only necessary when rolling up
+	 * individual chunks. In that case, insert_options must be 0.
```

Comment:
I forgot what a rollup is, is it when we merge an uncompressed chunk into a previously existing different compressed chunk? Is it the same when we merge the uncompressed part of partial chunk into the compressed one?

I don't understand the comment well, maybe it can be rephrased into something like: "If we are rolling up an uncompressed chunk into an existing compressed chunk, we have to fetch sequence numbers for each batch of the segment using the respective index. In normal compression, this is not necessary because we start segment numbers from zero. We distinguish these cases by insert_options being zero for rollups."